### PR TITLE
New cmd: query ibc-transfer escrow-address

### PR DESCRIPTION
**Pull request by ebuchman**
## Description

New command to get the escrow address for a channel.

Already merged into the cosmos-sdk  `release/v0.42.x` branch: https://github.com/cosmos/cosmos-sdk/pull/9383

closes: #189 

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: modules/apps/transfer/client/cli/query.go [hunk 1]
```diff
@@ -106,3 +106,28 @@ func GetCmdParams() *cobra.Command {
...
+		Args:    cobra.ExactArgs(2),
+		Example: fmt.Sprintf("%s query ibc-transfer escrow-address [port] [channel-id]", version.AppName),
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			clientCtx, err := client.GetClientQueryContext(cmd)
+			if err != nil {
+				return err
+			}
+			port := args[0]
+			channel := args[1]
+			addr := GetEscrowAddress(port, channel)
```
Comment (AdityaSripal) [inline modules/apps/transfer/client/cli/query.go:125]: How is this building? Shouldn't it be `types.GetEscrowAddress`?

Comment (AdityaSripal) [reply modules/apps/transfer/client/cli/query.go:125]: Ahh it isn't building

Comment (AdityaSripal) [inline modules/apps/transfer/client/cli/query.go:125]:
```suggestion
			addr := types.GetEscrowAddress(port, channel)
```

**Review by colin-axner — APPROVED** [review]
testedACK

```
simd q ibc-transfer escrow-address transfer channel-0
cosmos1a53udazy8ayufvy0s434pfwjcedzqv34kvz9tw
```

File: modules/apps/transfer/client/cli/query.go (hunk 1 above)
Comment (ebuchman) [reply modules/apps/transfer/client/cli/query.go]: my goodness I'm sorry about that, thanks for covering!

**Comment by colin-axner** [comment]
I'm not sure why I'm thinking of this now, but we should pass in a full path as opposed to a port and channel, otherwise this only supports single hops ie my path could be: 
`transfer/channel-1/transfer/channel-0`

We can fix on ibc-go, should we backport this? Seems like the primary usage in the short term is single hops?

**Pull request merged**